arrows/kpf/detected_object_set_output_kpf.cxx

```diff
@@ -148,6 +150,7 @@ write_set( const kwiver::vital::detected_object_set_sptr set, std::string const&
     w.set_schema( KPF::schema_style::GEOM )
       << KPF::writer< KPFC::kv_t >("detector_name", (*det)->detector_name())
       << KPF::writer< KPFC::id_t >(id, KPFC::id_t::DETECTION_ID)
+      << KPF::writer< KPFC::id_t >(id, KPFC::id_t::TRACK_ID)
```

Comment:
Just making sure I understand-- this is setting the track ID to be the same as the detection ID. A possible alternative would be to omit the track ID here and the reader would say "If the track ID is not otherwise set, use the detection ID."

Reply:
That wasn't the intent.  The goal is to set both DETECTION_ID and TRACK_ID to two different values?

Reply:
Determined that det_index, not id, should be written here; fixed in subsequent commit.